src/Symfony/Component/Console/Completion/CompletionInput.php

```diff
@@ -31,11 +31,11 @@ final class CompletionInput extends ArgvInput
     public const TYPE_OPTION_NAME = 'option_name';
     public const TYPE_NONE = 'none';
 
-    private $tokens;
-    private $currentIndex;
-    private $completionType;
-    private $completionName;
-    private $completionValue = '';
+    private ?array $tokens = null;
```

Comment:
Both construct methods pass an array, not null:
```suggestion
    private array $tokens;
```